src/components/Search/index.tsx

```diff
@@ -235,6 +239,10 @@ function Search({queryJSON, searchResults, onSearchListScroll, contentContainerS
     const [outstandingReportsByPolicyID] = useOnyx(ONYXKEYS.DERIVED.OUTSTANDING_REPORTS_BY_POLICY_ID, {canBeMissing: true});
     const [violations] = useOnyx(ONYXKEYS.COLLECTION.TRANSACTION_VIOLATIONS, {canBeMissing: true});
 
+    const [allPolicies] = useOnyx(ONYXKEYS.COLLECTION.POLICY, {canBeMissing: true});
```

Comment:
Use selector on this subscription and restrict it to `getActivePolicies(allPolicies ?? {}, currentUserLogin).filter((userPolicy) => isPaidGroupPolicyPolicyUtils(userPolicy))`

Reply:
Normally, you would use the other hook to get `currentUserLogin` and then use it in your selector and add `currentUserLogin` as dependency to `useOnyx` hook.